Tools/CISupport/ews-build/steps.py

```diff
@@ -7364,6 +7343,57 @@ def getResultSummary(self):
         return {u'step': status}
 
 
+class FindUnexpectedStaticAnalyzerResultsWithoutChange(FindUnexpectedStaticAnalyzerResults):
+    name = 'find-unexpected-results-without-change'
+
+    @defer.inlineCallbacks
+    def run(self):
```

Comment:
Please check if we can share more code especially for run() by moving it to base class entirely, and have run() call a method to calculate the self.command which is overridden in derived class. For eg, see `setLayoutTestCommand()` in this file.

If that's doesn't work, no worries.

Reply:
This may be difficult given the subtle differences in logic in what needs to be parsed after running the command, but I will explore it in the next PR.